data/transactions/logic/fields.go

```diff
@@ -1388,6 +1400,78 @@ var AcctParamsFields = FieldGroup{
 	acctParamsFieldSpecByName,
 }
 
+// VoterParamsField is an enum for `voter_params_get` opcode
+type VoterParamsField int
+
+const (
+	// VoterBalance is the balance, with pending rewards, from the balance
+	// round.  It is 0 if the account was offline then.
+	VoterBalance VoterParamsField = iota
+
+	// expose voter keys?
```

Comment:
I won't block PR on this, but assume this comment was left unintentionally.

Reply:
It's intentional, but really should be improved. There are some arguments for and against exposing them. The comment should really point them out.